deps/http_parser.mk

```diff
@@ -7,7 +7,7 @@ $(eval $(call git-external,http_parser,HTTPPARSER,,,$(BUILDDIR)))
 # Manually set the PLATFORM flag to whatever $(OS) is, because at least one
 # person (@staticfloat) has the PLATFORM envvar set to something else.
 # Also manually set PREFIX to DESTDIR
-HTTPPARSER_FLAGS := PLATFORM=$(shell echo $(OS) | tr A-Z a-z) PREFIX=$(build_prefix)
+HTTPPARSER_FLAGS := PLATFORM=$(shell echo $(OS) | tr A-Z a-z) PREFIX=$(build_prefix) LIBDIR=$(build_shlibdir)
```

Comment:
[They're not](https://github.com/nodejs/http-parser/blob/master/Makefile#L125-L128). 

Could `ln -s` be creating a nasty file on Windows?  Are we not allowed to have symlinks on Windows?  If not, my proposed solution is to just add an `rm libhttp_parser.dll` to the windows install steps and tell everyone to add the fully versioned name to any bindeps scripts that need it.